terraform/variables.tf

```diff
@@ -14,3 +14,12 @@ variable "common-tags" {
     "deploy-method" = "terraform"
   }
 }
+
+variable "cdn_stage_url" {
```

Comment:
Is it worth moving this to a local variable? I'm not sure if you would ever do a deploy of the cdn to a different endpoint without wanting to change the code. Being able to override this value just at apply time would be the main advantage of leaving it here.  (This https://github.com/mozmeao/infra-services/blob/master/gitlab-runners/tf/variables.tf plus https://github.com/mozmeao/infra-services/blob/master/gitlab-runners/tf/runner.tf#L86 as an example of what I'm suggesting).

Reply:
This is a good point. I struggle some times to decide if using local variables or not.
Probably I'd never want to modify this var during apply time, a part of it, what's the advantage you see in using a local var?

Reply:
Positives of the local var: prevents you from repeating the same url in multiple places in code.  Easy to change (change in variables file and plan/apply should work as normal). I choose between local and full variable based on when/if I need to change it at deploy time.  Otherwise they're almost the same.

Reply:
thanks for the feedback, I'll set it as local variable